### DISCUSSION TITLE
Game is minimized on stream but visible on host PC.

### DISCUSSION THREAD

**Discussion by DKogut17** [LizardByte/community#394] [Sunshine]
Hi, I'm having an issue where I cannot jump back into an active game that is open after disconnecting from a session for changing stream settings. In this scenario I'm playing Madden 24 on a Windows 10 host pc. I can launch the game from Moonlight successfully but when I am adjusting settings like lowering the bit rate, I jump back to "resume session" to the game it loads the stream back up but then the game flashes and disappears to appear to be minimized and I can only see my desktop. 

However, the input is still recognized and the audio is still on the game because I can hear it switching menus and all of that without issue. So I journey back to the host PC and the application is open and running just fine on my PC monitor but when I travel back to my TV I can see nothing but the desktop background and hear the game audio. The only workaround I have currently is to close the game out and start it all over. 

This is fine for having it set up after all the settings are perfect but I would love to understand this issue better because it seems like it's just a misconfiguration of some sort because otherwise it all works great.

Thank you!

**Comment by seismickarma** [comment]
Did you every find a solution to this?

**Comment by DKogut17** [comment]
no

**Comment by zorgick** [comment]
Try using virtual display, I had the same problem with chrome application, which was active on host, but not visible on the screen of the client. Display problem.

Enable while streaming a virtual display only, when you exit streaming enable phisical display.

**Reply by zorgick** [reply]
https://www.reddit.com/r/MoonlightStreaming/comments/1cytahf/guide_to_virtual_monitor_streaming_using/

[a comment by an automated account is not shown]
